doc/src/manual/functions.md

```diff
@@ -377,9 +397,9 @@ In all these cases, `x` is bound to a tuple of the trailing values passed to `ba
 It is possible to constrain the number of values passed as a variable argument; this will be discussed
 later in [Parametrically-constrained Varargs methods](@ref).
 
-On the flip side, it is often handy to "splat" the values contained in an iterable collection
-into a function call as individual arguments. To do this, one also uses `...` but in the function
-call instead:
+Alternatively, it is often handy to "splat" the values contained in an iterable collection
```

Comment:
This isn't really an alternative. It's call site vs definition site. I think `on the flip side` is fine.